utility/src/main/java/org/oppia/util/parser/JsonParser.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.parser
+
+import org.json.JSONArray
+
+/** This class parse json to mutable list of String. */
+class JsonParser {
+  fun parseJsonArrayToMutableListOfString(jsonArray: JSONArray): MutableList<String> {
+    val list: MutableList<String> = ArrayList()
+    for (i in 0..jsonArray!!.length() - 1) {
```

Comment:
Why is !! needed here? The jsonArray variable is non-null.

Reply:
Resolved